package.json

```diff
@@ -77,6 +77,7 @@
     "date-fns": "3.3.1",
     "ethers": "^6.10.0",
     "i18next": "23.8.2",
+    "matomo-types": "git+https://github.com/lukaw3d/matomo-types.git#018cf2cc8c62f7e467f88941725c93c369bffe3b",
```

Comment:
This would be nice, if it was a deployed npm package.

I see that this was forked, any reason why we can't directly reference the original package? Since we are only using types, it shouldn't affect the build.

Reply:
I dont think the fork is on npm + I had to fix some types. I'd like to publish this into @types/matomo, but I'm not sure how to correctly attribute+license it (DefinitelyTyped doesn't have license files per project)

Reply:
In that case I would rather see if the types are just copied inside the project to type definition file. And replaced with published npm package, once done.

Reply:
Yep, can do that

is there a security concern with github dependencies in npm?

Reply:
No, not really a security concern. But it is a bit inconsistent with other packages. Since it's only types, it shouldn't matter. But if it was code related, Renovate + Dependabot wouldn't pick up the package for example.